open_humans/templatetags/utilities.py

```diff
@@ -298,3 +299,37 @@ def add_string(a, b):
     Like `add` but coerces to strings instead of integers.
     """
     return str(a) + str(b)
+
+
+@register.tag
+def is_visible(parser, token):
```

Comment:
I think this is acting as a rendering filter than returning a boolean, is that right? if so, I wonder about renaming it to `render_if_visible`?